f/symfony/Makefile

```diff
@@ -1,5 +1,5 @@
 install:
-	composer install --classmap-authoritative --no-scripts --no-dev
+	composer install --optimize-autoloader --no-scripts --no-dev
```

Comment:
This shouldn't matter to much, but doesn't have a downside for dynamic classes.

Reply:
If we don't need this it's better to stay with the (slightly) faster alternative though.

https://getcomposer.org/doc/articles/autoloader-optimization.md

> The only issue is it does not keep track of autoload misses (i.e. when it can not find a given class), so those fallback to PSR-4 rules and can still result in slow filesystem checks.